Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -62,6 +70,12 @@ if(!$noDiscoveryService){
     Register-IdentityWithDiscovery -iisUserName $iisUser.UserName -metadataConnStr $metadataDatabase.DbConnectionString -version $installApplication.version -identityServerUrl $identityServiceUrl
 }
 
+$useAzure = $installSettings.useAzure
+if($null -eq $useAzure) {
+    $useAzure = $false
+    Add-InstallationSetting -configSection $installSettingsScope -configSetting "useAzure" -configValue "$useAzure" -installConfigPath $installConfigPath | Out-Null
```

Comment:
Same question as above...

Reply:
This one I feel should stay, if not here, somewhere else, since this will be configurable, it would be easier to toggle it going forward by setting the value to true/false instead of writing the whole element manually